hknweb/academics/models/course_surveys.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from .base_models import AcademicEntity
+
+
+class Question(AcademicEntity):
+    pass
+
+
+class Survey(AcademicEntity):
+    # reference attributes
+    survey_icsr = models.ForeignKey('ICSR', on_delete=models.CASCADE, related_name='survey_icsr')
+
+    # value attributes
+    num_students = models.IntegerField()
+    response_count = models.IntegerField()
+    is_private = models.BooleanField(default=True)
+
+
+class Rating(AcademicEntity):
+    # reference attributes
+    rating_question = models.ForeignKey('Question', on_delete=models.CASCADE, related_name='rating_question')
+    rating_survey = models.ForeignKey('Survey', on_delete=models.CASCADE, related_name='rating_survey')

```

Comment:
```suggestion
    question = models.ForeignKey('Question', on_delete=models.CASCADE)
    survey = models.ForeignKey('Survey', on_delete=models.CASCADE)
```

Otherwise code which uses ratings may end up looking like `rating.rating_question` and `rating.rating_survey`.

Reply:
Unfortunately, if we use a `rating.question`, Django considers that a clash with the `Question` class (for example). This is my workaround, any suggestions to the naming?

Reply:
I think that the slight amount of redundancy is ok; using a word other than "rating" for the prefix would just be confusing, and Brian mentioned the naming. Though I might be missing something, but I don't see why this would create a conflict.

Reply:
Leaving in for now; we can always revisit it.